src/Block/MediaBlockService.php

```diff
@@ -223,7 +223,7 @@ private function getMediaBuilder(AdminFormMapper $form): FormBuilderInterface
         ]);
 
         $formBuilder->addModelTransformer(new CallbackTransformer(
-            static fn (mixed $value): ?MediaInterface => $value,
+            static fn (MediaInterface $value): ?MediaInterface => $value,
```

Comment:
```suggestion
            static fn (?MediaInterface $value): ?MediaInterface => $value,
```
Null must be supported